recipes/python-cmethods/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "python_cmethods" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e03c78f610eedac605af1b60d58fdf81c1e9d7f1ee1b77b918e9256c71056ac2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - "cmethods = cmethods:cli"
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools_scm
+    - setuptools
+    - wheel
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - xarray >=2022.11.0
+    - netCDF4 >=1.6.1
+    - numpy
+    - cloup
+
+test:
+  imports:
+    - cmethods
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - cmethods --help
+about:
+  home: https://github.com/btschwertfeger/python-cmethods
+  summary: 'A collection of bias correction techniques written in Python - for climate sciences.'
+  description: |
+    Welcome to python-cmethods, a powerful Python package designed for bias
+    correction and adjustment of climate data. Built with a focus on ease of use
+    and efficiency, python-cmethods offers a comprehensive suite of functions
+    tailored for applying bias correction methods to climate model simulations
+    and observational datasets via command-line interface and API.
+
+    Please cite this project as described in
+    https://zenodo.org/doi/10.5281/zenodo.7652755.
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  doc_url: https://python-cmethods.readthedocs.io/en/stable/
+  dev_url: https://python-cmethods.readthedocs.io/en/latest/
```

Comment:
The `dev_url` is typically the github repo, so here it should be the same as the `home` url.